doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -38,6 +38,9 @@ You can also use FOTA updates to replace the application.
 FOTA over Bluetooth Low Energy
 ==============================
 
+Build configuration for MCUboot in the standard mode
```

Comment:
Sorry, I changed my mind. There are lots of labels in this section that are potentially used in other Working with guides. Let's remove this subsection as it can destroy this composability (we introduce the notion of the MCUboot configuration that is not present in the  fota_upgrades_over_ble_intro)

Reply:
removed

Reply:
Discussed offline, will revert direct-xip mode and testing steps subsections and keep this one (standard mode) removed